resources/mappings.json

```diff
@@ -0,0 +1,30 @@
+{
+   "doc":{
+      "fieldFormatMap": "{
+         \"Duration\": {
+            \"id\":\"number\",
+               \"params\": {
+                  \"pattern\":\"00:00:00\"
+               }
+         },
+         \"TotalBytesDelta\": {
+            \"id\": \"bytes\"
+         },
+         \"TotalBytes\": {
+            \"id\":\"bytes\"
+         },
+         \"SrcBytes\": {
+            \"id\":\"bytes\"
```

Comment:
from Kjell: 

FYI for the reviewers. Internal Kibana magic will make the "bytes" field automatically transform the value into appropriate byte-type. I.e. TB, GB, MB, KB etc.